app/services/bryant_street_studios_webhook.rb

```diff
@@ -0,0 +1,23 @@
+class BryantStreetStudiosWebhook
+  def self.artist_updated(artist_id)
```

Comment:
Maybe down the road we filter this by artists who are at 1890.  b/c right now any artist update will ping. 1890 will happily delete nothing from the cache but it's more traffic than we actually need.

Reply:
Oh yeah that totally makes sense, so we should have a studio check